### DISCUSSION TITLE
from mycroft.util.parse import extract_numbers    - what is the OVOS replacement

### DISCUSSION THREAD

**Discussion by geetee24** [OpenVoiceOS/ovos-buildroot#229] [Q&A] [answered]
I use from mycroft.util.parse import extract_numbers

what is the lib from ovos?

**Comment by JarbasAl** [comment, marked as answer]
`from lingua_franca.parse import extract_numbers`

this comes from https://github.com/OpenVoiceOS/ovos-lingua-franca , it was the same as mycroft, but mycroft had the same import duplicated under `mycroft.util`
